app/services/harmony_client.rb

```diff
@@ -21,7 +21,16 @@
     ServiceClient::Middleware::Timing.new,
     ServiceClient::Middleware::BodyEncoder.new(:transit_msgpack),
     ServiceClient::Middleware::ParamEncoder.new,
-    ServiceClient::Middleware::JwtAuthenticator.new(APP_CONFIG.harmony_api_disable_authentication,
-                                                    APP_CONFIG.harmony_api_token_secret),
+    ServiceClient::Middleware::JwtAuthenticator.new(disable: APP_CONFIG.harmony_api_disable_authentication,
```

Comment:
How about an enter before `disable: ...`, would keep the line length in control.